lib/rootstrap.rb

```diff
@@ -1,6 +1,15 @@
+# frozen_string_literal: true
+
 require 'rootstrap/version'
+require 'rootstrap/gem'
+require 'rootstrap/context'
+require 'rootstrap/tasks/add_dependency'
+require 'rootstrap/dependencies/base_dependency'
+require 'rootstrap/dependencies/rake'
+require 'rootstrap/dependencies/reek'
+require 'rootstrap/dependencies/rspec'
+require 'rootstrap/dependencies/rubocop'
+require 'rootstrap/dependencies/simplecov'
 
 module Rootstrap
```

Comment:
I don't like using Rootstrap as the namespace, I'd rather go with `RSGemGenerator` or something like that